### ISSUE TITLE
Build fails in paths with spaces

### ISSUE THREAD

**Issue by julfersnlt** [LEDApplications/lehd-schema#4]
The script `generate_schema_docs.sh` fails when this project is checked out into a directory that includes spaces:

```
./generate_schema_docs.sh: line 105: cd: too many arguments
asciidoctor: FAILED: input file [path fragment]/csvsubcolumn-include-processor.rb is missing
asciidoctor: FAILED: input file [path fragment]/csvsubcolumn-include-processor.rb is missing
asciidoctor: FAILED: input file [path fragment]/csvsubcolumn-include-processor.rb is missing
asciidoctor: FAILED: input file [path fragment]/csvsubcolumn-include-processor.rb is missing
```

Paths in the script should be quoted.

**Comment by julfersnlt** [comment]
I can send a pull request for this

**Comment by jodyhoonstarr** [comment]
Closed by #6
